test/e2e/windows/dns.go

```diff
@@ -50,6 +50,9 @@ var _ = SIGDescribe("DNS", func() {
 			Nameservers: []string{testInjectedIP},
 			Searches:    []string{testSearchPath},
 		}
+		testUtilsPod.Spec.NodeSelector = map[string]string{
```

Comment:
this was added in https://github.com/kubernetes/kubernetes/pull/97804

Reply:
yup rebased just now